classes/app.rb

```diff
@@ -0,0 +1,182 @@
+require './classes/book'
+require_relative './music_album'
+require_relative './genre'
+require './classes/label'
+require './classes/game'
+require './classes/author'
+require './modules/preserver_module'
```

Comment:
- [ ] I would recommend using `require_relative` instead of `require` if you are importing from the same repo. `require` is basically used when we import any `library` or `gem` from outside the repo. Please follow these best practices in this and every file.